engines/standard_tasks/spec/support/pages/overlays/register_decision.rb

```diff
@@ -15,4 +15,10 @@ def decision_letter
   def accepted?
     find('input[value="accepted"]')
   end
+
+  def disabled?
+    find("#task_completed[disabled]") != nil &&
+    all("input[type='radio'][disabled]").size == 3 &&
+    find("textarea[disabled]") != nil
```

Comment:
Use 2 (not 0) spaces for indenting an expression spanning multiple lines.<br>Prefer `!expression.nil?` over `expression != nil`.